server/src/configs/default.json

```diff
@@ -75,13 +75,18 @@
         "sm": 20,
         "md": 35,
         "lg": 50,
-        "xl": 65
+        "xl": 65,
+        "offsetX": 0,
+        "offsetY": 0
```

Comment:
We should add offsets to the specific icon packs instead of in the universal sizes section. This will be different for every icon set, which can be changed on the client side so even if you get these dialed in perfectly for whatever set *you* prefer, maybe your users prefer the other icons and will now not be aligned properly.